estuary_updater/handlers/base.py

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: GPL-3.0+
 
-from __future__ import unicode_literals
+from __future__ import unicode_literals, absolute_import
```

Comment:
For consistency in the project, could you file a separate PR with this import added to all of our files? That way one file doesn't import differently than the rest.